.github/actions/telegram-status-report/reporter.sh

```
@@ -0,0 +1,41 @@
+Logs: https://github.com/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}"
+
+# send to telegram
+RES=$(curl -s -X POST "https://api.telegram.org/bot${TELEGRAM_BOT_TOKEN}/sendMessage" \
+    -d "chat_id=${TELEGRAM_CHAT_ID}" \
+    -d "parse_mode=Markdown" \
+    -d "text=$TEXT")
+
+# check if sent successfully
+if echo "$RES" | grep -q '"ok":false'; then
+    echo "❌ Failed to send Telegram message"
+    echo $RES
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)